components/centraldashboard/app/api.ts

```diff
@@ -3,24 +3,47 @@ import express from 'express';
 
 import {KubernetesService} from './k8s_service';
 
+interface PlatformStatics {
+    [key: string]: PlatformMeta;
+}
+interface PlatformMeta {
+    logo: string;
+    title: string;
+}
+
+/** API routes exposing information from Kubernetes */
+function _platformStatics(platform: string): {meta: PlatformMeta|undefined} {
+    console.log("Loading meta for", {platform});
+    const meta = ({
+        gce: {
+            logo: 'https://cloud.google.com/_static/images/cloud/icons/favicons/onecloud/super_cloud.png',
```

Comment:
One reason I had thought this would live in the client-side code was that we would package this image with our code and inline it like we do for the Kubeflow logo itself.